test/integration/outreach/outreach.js

```diff
@@ -308,6 +308,96 @@ context('Outreach', function() {
       .contains('Form Name');
   });
 
+  specify('User verification - user entered an invalid code', function() {
+    cy
+      .intercept('GET', '/api/outreach?filter[action]=11111', req => {
+        req.reply({
+          statusCode: 200,
+          body: {
+            data: {
+              attributes: {
+                phone_end: '1234',
+              },
+              relationships: {
+                patient: {
+                  data: {
+                    id: '1',
+                  },
+                },
+              },
+            },
+          },
+        });
+      })
+      .visit('/outreach/11111', { noWait: true, isRoot: true });
+
+    cy
+      .intercept('POST', '/api/outreach/1', {
+        delay: 100,
+        body: { data: {} },
+      })
+      .as('routeCreateVerifyCodeRequest');
+
+    cy
+      .get('.js-submit')
+      .click();
+
+    cy
+      .wait('@routeCreateVerifyCodeRequest');
+
+    cy
+      .get('.verify__code-fields')
+      .find('.js-input')
+      .first()
+      .type('5678');
+
+    cy
+      .intercept('POST', '/api/outreach/auth', {
+        statusCode: 403,
+        delay: 100,
+        body: {
+          data: {
+            patientId: '1',
+            opt: '5678',
+            attributes: {},
+          },
+        },
+      })
+      .as('routeVerifyCodeRequest');
+
+    cy
+      .get('.js-submit')
+      .click()
+      .should('be.disabled');
+
+    cy
+      .wait('@routeVerifyCodeRequest');
+
+    cy
+      .get('.verify__code-fields')
+      .find('.js-input.has-error')
+      .should('have.length', 4);
+
+    cy
+      .get('.verify__error-text')
+      .should('exist');
+
+    cy
+      .get('.verify__code-fields')
+      .find('.js-input')
+      .first()
+      .type('1');
+
+    cy
+      .get('.verify__code-fields')
+      .find('.js-input.has-error')
+      .should('not.exist');
+
+    cy
+      .get('.verify__error-text')
+      .should('not.exist');
+  });
```

Comment:
When the user enters an invalid OTP code:

1. The four input elements should have a red border.
2. The input fields should be cleared.
3. The `Incorrect verification code. Please try again.` error message should be shown.

When the user types into one of the fields again, the red input border and error messaging should be removed.

[Screenshot](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/2937c913-536e-4947-b10b-4258a3cf41dc)